### PR TITLE
change image cover to background-image

### PR DESCRIPTION
**Pull request by minhhvu**
it makes the images on sign up/log in more responsive. Let try on your screen and give me feedbacks

### CODE REVIEW

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ const Login = () => {
 
     return (
         <Grid container className={classes.vh100}>
```
Comment (aslamm29) [inline client/src/pages/Login.js:64]: as the screen resizes, the container changes to a vertical view from a horizontal view, can you try using flex direction or display inline-block to tackle it

File: client/src/pages/Signup.js [hunk 2]
```diff
@@ -40,8 +40,8 @@ const Signup = () => {
 
     return (
         <Grid container className={classes.vh100}>
```
Comment (aslamm29) [inline client/src/pages/Signup.js:43]: as the screen resizes, the container changes to a vertical view from a horizontal view, can you try using flex direction or display inline-block to tackle it

**Review by aslamm29 — CHANGES_REQUESTED** [review]

File: client/src/pages/Signup.js (hunk 2 above)
Comment (minhhvu) [reply client/src/pages/Signup.js:43]: you mean to change the Grid into flex/inline-bloack??

Comment (gonzalj3) [reply client/src/pages/Signup.js:43]: I dont think this is a bug. I think all react apps work this way. It makes sense for it to switch from horizontal to vertical. My thinking is where would the image go if the container did not switch from horizontal to vertical? There is no other space on  the screen, like on a phone.

**Review by gonzalj3 — APPROVED** [review]
I think the image is working fine at this point.

File: client/src/pages/Signup.js (hunk 2 above)
Comment (aeciorc) [reply client/src/pages/Signup.js:43]: Yes,  I agree. It should turn to vertical on small screen

**Review by aeciorc — APPROVED** [review]

**Review by aslamm29 — APPROVED** [review]

**Pull request merged**